dialects/math/math.thorin

```diff
@@ -153,6 +154,25 @@
 ///
 .ax %math.gamma(t, l): Π.[pe: «2; .Nat»][.Nat][%math.F pe] -> %math.F pe, normalize_gamma;
 ///
+/// ### %math.abs
+///
+/// [Absolute value] (https://en.wikipedia.org/wiki/Absolute_value) Returns absolute value of the provided value
```

Comment:
The markdown would render as:
"Absolute value Returns the absolute value of the provided value."
https://anydsl.github.io/thorin2/math.html#autotoc_md132

Maybe
`Returns [absolute value](...) of the provided value.`
or
`[Absolute value](...) of a an floating point` 
would fit better

(Also, I am not sure if markdown can handle the space between the text and the hyperlink.)

Reply:
Yeah, I've already the errors in the markdown. Will fix them